### ISSUE TITLE
Feedback course 09: Peer Review

### ISSUE THREAD

**Issue by christian-rli** [OpenEnergyPlatform/examples#189]
Please review course 09 : https://openenergyplatform.github.io/academy/courses/09_peer_review/
and provide your feedback as comments below. Feedback can include pointing out errors (such as broken links, things that do not work as described) any suggestions for improvements, things that you expected but missed, etc.

In your comment, please make sure to reference the material you're referring to (the courses link to various materials)

**Comment by Ludee** [comment]
The current version is a technical documentation and not a course.
Move the existing text to compendium and create a training course.

**Comment by han-f** [comment]
Would it be possible to add a hands-on version of this course that teaches how to open peer review on the OEP without the functionality being completely finished and then update in case we have updates? I

**Comment by jh-RLI** [comment]
Yes, this is possible and is much needed. I think the current course is more of a high level description, more theoretical, but it also explains important parts of our specific peer review. 

@bmlancien and I are currently working on creating an overview picture of the peer review process, which should be a good starting point to explain in detail how peer review works. The ‘user guide’ part, where we show screenshots and explain how to do the open peer review in the OEP, is definitely one thing we need to add. I'm wondering if we should put it all in one course or split it into a theoretical and a practical part?

**Comment by han-f** [comment]
That sounds very good @jh-RLI  and @bmlancien. My personal feeling would be to change the current course into a real hands-on course on how to do it on the OEP, and to link to another resource (legacy content of current course, or external resource) on open peer review. That way a user will be enabled quickly to work with OEP functionalities and can decide whether he/she needs more theoretical inputs.

A good looking external source is this one: https://www.fosteropenscience.eu/learning/open-peer-review/#/id/5a17e150c2af651d1e3b1bce

**Comment by jh-RLI** [comment]
Okay, thanks, I agree, I also think it's better this way = focus on enabling users to use our actual features. That is great link to the external source.

By the way, a general question: would you want to see an overview like this one at the beginning of the course or would you feel disturbed by the complexity / it is too technicall ? :) 

The image is not yet finalized: it´s still quite full and it needs textual explanation for some elements (e.g. autoamtic review, when is it complete, how to start another one), I think we need to reduce it a bit, but it should still help to understand the hole process.

![image](https://github.com/OpenEnergyPlatform/academy/assets/38939526/94969885-3b2c-4f7f-89da-c077e50ab7d4)

**Comment by han-f** [comment]
I like the figure even if it is complex. I am not sure whether a placement at the beginning or the end would be best. If the course addresses the steps of the process along the figure, it may serve well as a summary at the end and as a "map" for people who want to conduct a review...
